pkg/monitoring/metrics/virt-controller/vmistats_collector.go

```diff
@@ -160,7 +161,7 @@ func getGuestOSInfo(vmi *k6tv1.VirtualMachineInstance) (kernelRelease, machine,
 	}
 
 	if vmi.Status.GuestOSInfo.Machine != "" {
-		machine = vmi.Status.GuestOSInfo.Machine
+		guestOSMachineArch = vmi.Status.GuestOSInfo.Machine
```

Comment:
we are keeping the "guest_os_machine" and added "guest_os_arch" with the same values, is this intentional? some concern about compatibility?